server/models/user.js

```diff
@@ -32,6 +32,14 @@ function constructor(...args) {
       email.value = email.value.toLowerCase();
     }
   });
+
+  this.on('fetching:collection', (model, columns, opts) => {
```

Comment:
This could be simplified into a single function called for collection:fetching and fetching.
However, it's very unlikely to change, so the gain is minimal